src/mongodb/index.ts

```diff
@@ -0,0 +1,7 @@
+const uri = "mongodb+srv://ivan_kuxta:ivan_kuxta@learnnodejs.nlxd0u0.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
Please, use .env file for sensitive data like this.

Reply:
thanks for the feedback, I changed the previous pull, but didn't change this one